pallets/moonbeam-foreign-assets/src/tests.rs

```diff
@@ -422,3 +422,47 @@ fn test_governance_can_change_any_asset_location() {
 		));
 	});
 }
+
+#[test]
+fn test_inspect_trait_impl() {
```

Comment:
can we add more tests to test other functions in `impl_fungibles.rs`?

Reply:
will do

Reply:
amended [d3c320b](https://github.com/moonbeam-foundation/moonbeam/pull/3220/commits/d3c320bec5785f9ae7eef17e9a1bcd0858f9a861)